examples/multus.yml

```diff
@@ -0,0 +1,134 @@
+---
```

Comment:
why do we need this file under our examples? how is it different than the one taken from official multus site?

Reply:
For OpenShift, changed its name.